web/src/locales/en-US/component.ts

```diff
@@ -51,6 +51,8 @@ export default {
   'component.global.pleaseEnter': 'Please Enter',
   'component.global.pleaseChoose': 'Please Choose',
   'component.global.pleaseCheck': 'Please Check',
+  'component.global.pleaseSelect': 'Please Select',
```

Comment:
Hi @juzhiyuan and @LiteSun 

I found here, we have the `choose` and `select`, I think we can choose only one?

Reply:
Agree with you.